src/tribler/core/components/tag/db/tag_db.py

```diff
@@ -19,6 +18,32 @@
 HIDE_THRESHOLD = -2
 
 
+class Operation(IntEnum):
+    ADD = 1
+    REMOVE = 2
+
+
+class Predicate(IntEnum):
+    CONTRIBUTOR = 1
+    COVERAGE = 2
+    CREATOR = 3
+    DATE = 4
+    DESCRIPTION = 5
+    FORMAT = 6
+    IDENTIFIER = 7
+    LANGUAGE = 8
+    PUBLISHER = 9
+    RELATION = 10
+    RIGHTS = 11
+    SOURCE = 12
+    SUBJECT = 13
+    TITLE = 14
```

Comment:
Do we need the `TITLE` predicate? The title should already be included in the subject field of a triple.